src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Helper executable: generates parts of the sleef header file
+add_executable(${TARGET_MKRENAME} mkrename.c)
+target_compile_options(${TARGET_MKRENAME}
+  PRIVATE ${FLAGS_WALL})
+
+# TODO: set the parameters for mkrename run for the SIMD implementation
+set(MKRENAME_PARAMS "")
+
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/GenerateSleefHeader.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake @ONLY)
+
+# Add custom command to generate the sleef header at build time
+add_custom_command(
+  OUTPUT ${CMAKE_BINARY_DIR}/include/sleef.h
+  COMMAND ${CMAKE_COMMAND} -P
+    ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake
+  DEPENDS
+    ${CMAKE_CURRENT_SOURCE_DIR}/sleeflibm.h.org
+    ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake)
+
+# Add target for generating the sleef header
+add_custom_target(generate-header ALL
+  DEPENDS ${CMAKE_BINARY_DIR}/include/sleef.h)
```

Comment:
I think this is not needed. The custom_command dependency is already specified in all targets when you use `${CMAKE_BINARY_DIR}/include/sleef.h`. Please remove.